argus/config_generator/windows/base.py

```diff
@@ -59,7 +64,8 @@ def _get_base_conf(config_name):
         base_conf = StringIO.StringIO(
             util.get_resource(config_name))
         conf = six.moves.configparser.ConfigParser()
-        conf.readfp(base_conf)
+        # pylint: disable=deprecated-method
+        conf.read(base_conf)
```

Comment:
Read should only be called after `readfp`, please change it back to `readfp` and find another away if possible to pass the `deprecated-method` warning.